scripts/pyvenv_requirements.txt

```diff
@@ -0,0 +1,17 @@
+anyconfig==0.9.11
+attrs==20.2.0
+click==7.1.2
+et-xmlfile==1.0.1
+fastjsonschema==2.14.5
+jdcal==1.4.1
+Jinja2==2.11.2
+jsonnet==0.16.0
+jsonpointer==2.0
+jsonschema==3.2.0
+MarkupSafe==1.1.1
+numpy==1.19.4
+openpyxl==3.0.5
+pyrsistent==0.17.3
+six==1.15.0
+# Comment out moo, as by default PyPI's moo will be picked up.
+# moo==0.2.0
```

Comment:
You might be able to replace these lines with one like:
```
git+git://github.com/brettviren/moo.git@0.3.0#egg=moo
```

Reply:
Thanks for the suggestion. I tried it in this commit https://github.com/DUNE-DAQ/daq-buildtools/pull/51/commits/4ec2be3a4aa445dfec082eb79352fb81e22024f7. 

But it looks like pip was still checking the moo in PyPI. I got the error messages as the following:
```shell
  ERROR: Could not find a version that satisfies the requirement moo==git+https://github.com/brettviren/moo.git@0.3.0#egg=moo (from -r pyvenv_requirements.txt (line 17)) (from versions: 0.1.0, 0.1.1, 0.1.2, 0.1.5, 0.1.7, 0.1.8, 0.2.0, 0.5.3, 0.5.4, 0.5.5)
  ERROR: No matching distribution found for moo==git+https://github.com/brettviren/moo.git@0.3.0#egg=moo (from -r pyvenv_requirements.txt (line 17))      
```